app/models/PostrunAction.scala

```diff
@@ -92,14 +93,14 @@ case class PostrunAction (id:Option[Int],runnable:String, title:String, descript
         logger.debug(s"Going to try to run script at path $inputPath...")
         val scriptArgs = Map(
           "projectFile" -> projectFileName
-        ) ++ projectEntry.asStringMap ++ projectType.asStringMap
+        ) ++ projectEntry.asStringMap ++ projectType.asStringMap ++ commissionMaybe.map(_.asStringMap).getOrElse(Map()) ++ workingGroupMaybe.map(_.asStringMap).getOrElse(Map())
 
         runner.runScriptAsync(inputPath.toString, scriptArgs, dataCache) map {
           case Success(scriptOutput) =>
             logger.debug("Script started successfully")
             scriptOutput.raisedError match {
               case Some(error)=>
-                logger.error("Postrun script could not complete: ",error)
```

Comment:
this stack trace is never useful as it just has a "null" message and a ton of Jython stack frames. The actual python exception (captured in the stderr) is much more useful.